### DISCUSSION TITLE
Sharing github actions between repositories

### DISCUSSION THREAD

**Discussion by jessie828** [community/community#56823] [Actions]
I have about 4 repositories of which I would like to share github actions instead of copying them across each repository.

I have read it could be possible to create a ".github"repository in your organization but this will not work because I am part of an organisation of about 10k devs and 7k repositories and would only like to share the action with the 4 repos I am responsible for.

Now would it be possible I create a new repo called l "my-github-actions" and include that one as a git submodule
As a directory called .github in my 4 separate repositories?

**Comment by airtower-luna** [comment]
Having `.github` in a submodule won't work. Having custom actions or reusable workflows in a separate repository and calling them from your other repositories will (assuming they are accessible, see [Sharing actions and workflows with your organization](https://docs.github.com/en/actions/creating-actions/sharing-actions-and-workflows-with-your-organization) if you don't want to make them public).
